app/src/release/google-services.json

```diff
@@ -0,0 +1,40 @@
+{
+  "project_info": {
```

Comment:
I'm a little confused, should this have production/release info in it? Or is that all filled in later/deploy time?

Reply:
yeah this is replaced with the correct json config file when building the apk

Reply:
👍